create_graphics.py

```diff
@@ -447,10 +448,13 @@ def graphics_driver(cla):
     '''
 
     # Create an empty zip file
-    zipf = None
-    if cla.zip_dir:
-        os.makedirs(cla.zip_dir, exist_ok=True)
-        zipf = os.path.join(cla.zip_dir, 'files.zip')
+    zipf = []
+    for tile in cla.tiles:
+        if cla.zip_dir:
```

Comment:
Lines 452 and 453 should be in reverse order for efficiency purposes. Only go through the loop if zip_dir is defined.